shared/src/commonMain/kotlin/co/nimblehq/blisskmmic/data/model/SurveyDetailApiModel.kt

```diff
@@ -0,0 +1,30 @@
+package co.nimblehq.blisskmmic.data.model
+
+import co.nimblehq.blisskmmic.domain.model.SurveyDetail
+import co.nimblehq.blisskmmic.domain.model.Token
+import kotlinx.serialization.SerialName
+import kotlinx.serialization.Serializable
+
+@Serializable
+data class SurveyDetailApiModel(
+    val title: String,
+    val description: String,
+    @SerialName("is_active") val isActive: Boolean,
+    @SerialName("cover_image_url") val coverImageUrl: String,
+    @SerialName("survey_type") val surveyType: String,
+    val questions: List<SurveyIncluded>
+    ) {
```

Comment:
I think there is a indentation issue here 🤔
```suggestion
data class SurveyDetailApiModel(
    val title: String,
    val description: String,
    @SerialName("is_active") val isActive: Boolean,
    @SerialName("cover_image_url") val coverImageUrl: String,
    @SerialName("survey_type") val surveyType: String,
    val questions: List<SurveyIncluded>
) {
```